pyproject.toml

```diff
@@ -1,3 +1,88 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+
+[tool.poetry]
+name = "qcelemental"
+version = "0.25.1"
```

Comment:
So now version is decoupled from `git describe` and tags? Possibly that's ok, as we don't end up testing or pinning on non-tagged versions, but it's a little awkward to not be able to tell clean/dirty installations. Also now multiple sources of truth between pyproject.toml and git tags. What was the reasoning behind this vs. versioneer vs. one of https://github.com/python-versioneer/python-versioneer#similar-projects ?

Reply:
Complexity and standard practice. `versioneer` is leaning hard on the single-source-of-truth principle and I like where they're head is at, but it's over-engineered and unnecessary IMO. `Versioneer` been around for 12 years, yet has <1k stars--it's never really taken off. We were seeing bugs on the test suite from _version_ tests. We even had to write version tests! That's a lot of overhead for return a string. Even large python projects get by with a simple string in `__init__.py` (or `pyproject.toml`). I figured we would too :)